incubator/azuremonitor-containers/templates/omsagent-daemonset.yaml

```diff
@@ -98,6 +103,16 @@ spec:
         periodSeconds: 60
    nodeSelector:
     beta.kubernetes.io/os: linux
+   affinity:
```

Comment:
Why don't you make this configurable?

Reply:
> Why don't you make this configurable?

Hi, @lachie83, can you please provide more details what do you meant "configurable" here?  Since our agent doesnt support on virtual-kubelet(s) so this node affinity "not to get scheduled on to the virtual-kubelet nodes". The reason,  I brought this to have consistency with our chart in AKS.